### ISSUE TITLE
Announce information about sorting ability for user when aria-sort has value none 

### ISSUE THREAD

**Issue by kolaps33** [nvaccess/nvda#12806]
When there is in header grid aria-sort attribute with "none" value, then NVDA doesn't announce any information.

Based on aria-example
https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html#
There is recommendation role="none" when column is sortable but not sorted.
But in given example NVDA doesn't narrate to the screen reader user any information about ability of sort the column. 

### Describe the solution you'd like
Would be great to to narrate something like "sortable", when aria-sort="none"

### Describe alternatives you've considered
if screen reader doesn't implement, then everyone who use it need to custom solution

### Additional context
I find already discussion in JAWS backlog.
https://github.com/w3c/aria-practices/issues/1192

**Comment by feerrenrut** [comment]
This would also imply that the UX is the same for all sortable data grids. That seems like a big assumptions to make. 

Note that `aria-sort` can be used to indicate that data is already sorted, there may be no click handler on the heading, the user may not be able to change how the data is sorted. If NVDA announced "sortable", it would be confusing in the first situation. This is overloading the semantics of the attribute.

**Comment by Adriani90** [comment]
Given the comment by Reef above, I am closing as won't fix.
